### DISCUSSION TITLE
Repository Moved

### DISCUSSION THREAD

**Discussion by bjorn** [mapeditor/tiled-extensions#23] [Team Posts]
Thanks to @MikeMnD for the work so far on this extension and for transferring it to the @mapeditor organization! And thanks to @mcarpenter622, @agmcleod and @eleniums for joining the further maintenance of this extension!

The main goal with this move is to ensure any contributions can get a review from people who have contributed before and are users of this extension as well. It would also be nice, to give this extension more complete support for the features of both Tiled and Godot, but that's up to you depending on your interest and time. Please create issues regarding any ideas for improvement.

To ensure a similar quality to when @MikeMnD was reviewing each contribution, I've protected the `master` branch, which now requires a pull request with approval from at least one co-maintainer for any change. This has worked well for https://github.com/mapeditor/rs-tiled, which has also a few maintainers now.

**Comment by mcarpenter622** [comment]
Nice to meet you! Do you go by Thorbjorn or Bjorn? I'll be honest I don't know if your name should be shortened like that or not =).  I'm glad you protected the master branch that seems like a very reasonable requirement for changes.

Mark

**Comment by bjorn** [comment]
@mcarpenter622 Usually I'm just Bjørn, but I use my full name for disambiguating when necessary. That also affects usernames. :-)

**Comment by mcarpenter622** [comment]
Hey thanks for the clarification! =)
